src/pages/Cinemas/CinemasContainer.jsx

```diff
@@ -41,6 +50,17 @@ const CinemasContainer = () => {
       cinemasList.filter((cinema) => cinema.city._id === event.target.value)
     );
   };
+
+  const handleDeleteCinema = async (cinema) => {
+    const token = localStorage.getItem("token");
+    try {
+      await formatRequest("/cinema", "DELETE", token, { id: cinema._id });
```

Comment:
it's better to work on a token in `formatRequest` instead of duplicating the code